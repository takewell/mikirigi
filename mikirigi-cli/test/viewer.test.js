const { execQuery } = require("../src/lib/execQuery");
const { ViewerQuery, GetRepositoryStars } = require("../src/lib/query");

test("ViewerQuery", async () => {
  const res = await execQuery(ViewerQuery);
  expect(res).toMatchSnapshot();
});
