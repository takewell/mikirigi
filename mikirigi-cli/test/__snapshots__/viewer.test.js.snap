// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ViewerQuery 1`] = `
Object {
  "data": Object {
    "viewer": Object {
      "__typename": "User",
      "login": "takewell",
    },
  },
  "loading": false,
  "networkStatus": 7,
  "stale": false,
}
`;
